Code/PgSQL/rdkit/expected/rdkit-91.out

```diff
@@ -771,3 +771,77 @@ select 'C1CC1C(C)OC'::mol @> mol_adjust_query_properties('C1CC1CO*'::mol,'{"adju
  f
 (1 row)
 
+select 'C1CC1CCCC'::mol @> mol_adjust_query_properties('C1CC1CC'::mol);
```

Comment:
Can mol here be a ctab?  If so, we should probably aromatize by default using the new adjustQuery prop?  this actually might be true of smiles as well actually.